storops/unity/resource/host.py

```diff
@@ -238,6 +239,21 @@ def _attach_with_retry(self, lun_or_snap, skip_hlu_0):
         else:
             return host_lun.hlu
 
+    @version('>=4.4.0')  # noqa
+    @retry(limit=5, on_error=ex.UnityHluNumberInUseError)
+    def _attach_with_retry(self, lun_or_snap, skip_hlu_0):
+        # From 4.4.0 (Osprey), it supported to pass in hlu when attaching LUN,
+        # but not when attaching snap.
+        from storops.unity.resource.lun import UnityLun as _UnityLun
```

Comment:
Any reason to put it here instead of the head?

Reply:
Will cause import cycle then.